gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+
+## Problem
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other and connect.
+
+## Story (Optional)
+Your InnerSource program has been established and there is great interest but little actual activity. 
+Individual project owners are encountering difficulties attracting a community of contributors.
+Most managers are supportive in allowing their employees the time to involve themselves in an InnerSource project. 
+There simply is no single uniform way for contributors to discover all the InnerSource opportunities that exist that they might be interested in contributing to.
+Conversely, innerSource project owners also have no way of informing a general audience about their projects and its goals.
+
+## Context
+* There are multiple business units in your organization that do not communicate well.
+* Individuals have no way of discovering what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no way of advertising their project to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* Managers have no way of knowing what the true time/effort commitment will be if their employee chooses to take on tasks to support an InnerSource project.
+* There is no way of tracking or rewarding an individual's involvement in an InnerSource project.
+
+## Forces
+* Individuals are trapped within their business units/teams and do not often interact with those outside of them
+* Individuals are experiencing downtime on projects where they are underutilized
+* Managers are reluctant to allow involvement in work outside of their business unit priorities
+* Managers have no way of understanding what the time/effort commitment is for their employees wanting to get involved in an InnerSource project outside of their business unit
+* Managers have no way of tracking or rewarding their employee's involvement in InnerSource projects
+* There is no single portal through which InnerSource project owners can advertise the existence and goals of their projects
+* There is no single portal through which potential InnerSource contributors can discover project opportunities
+
+## Solutions
+Several separate initiatives were spawned across business units and then brought together by the InnerSource governance team to create an intranet based InnerSource marketplace.
+
+The first was an InnerSource project portal. This was an intranet site where InnerSource project owners could plug in the URL to their Git repo and (via the inclusion of an “innersource.json” file to their codebase) display small cards of information describing their projects. Each of these were then indexed and made searchable by end users based on a number of criteria including business unit name, technologies in use or trusted contributor names, etc.
```

Comment:
This is cool, but for the **Gig Marketplace** pattern specifically I think we should omit this first paragraph and just focus on the second.

Reply:
Agreed.  We will strike it out entirely.

Reply:
Good point.

Reply:
from @dicortazar : we are using the specific word _marketplace_ but there's no mention of _selling_ anthing
from @stephenmccall : what are the _buyers_ and _sellers_?
from @dicortazar : different BUs are offering time, capacity or money (sellers) and the developers (BU) could use that (buyers). How are the "selling" BUs reimbursed for their offering?
from @stephenmccall : the reward is being able to say "I did this". There are no physical rewards attached to work done. 
from @dicortazar : the developers are being rewarded by their organization. It would be a good idea for the selling business units to be reimbursed by the "buying" business units. 
from @stephenmccall : this should happen at some point, but we wanted to start this as "pro bono" to get things going. Might evolve to Daniels suggestion.
from @rshanmer : related to the performance management section. There are three parties: management rewarding their developers for doing work for a 3rd party (as part of performance management). I feel this constitutes a marketplace.
from @stephenmccall : there is inherent benefit in doing the work itself. Unsure whether there need to be additional benefits
from @NewMexicoKid : is it also a _people marketplace_? A tool for BUs to find people with certain skills? What implications might that have for the "buying" business units? (loosing their best people?)
from (unattributed): Talent hoarding can be a problem in companies; people's mobility around the company is limited. There was a program to solve this: You can allow people to secretly share their resume. This can be tied into this same idea.